scripts/all_signs.py

```diff
@@ -0,0 +1,75 @@
+#!/usr/bin/env python3
+
+#all_signs.py
```

Comment:
apparently you add a new script all_sign, while there is still all-sign around. Any reason to rename it?

Reply:
It is Pierre's script.

Reply:
what is it for? should it replace the other one which has almost the same file name?